onboarding.md

```diff
@@ -0,0 +1,28 @@
+# Onboarding
+
+All participants in the OpenJS Foundation projects and groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md).
+There are further expectations for members who represent the Standards Working Group (hereby called representatives).
+
+It is understood that representatives will be a part of individual or their company specific projects. But while representing
+the Standards Working Group, they should take decisions/discuss/promote in the interest of Standards Group. 
+
+If there is a conflict (due to personal or company specific projects), then the representative should be explicit 
+about the hat they are wearing while deciding/discussing/promoting the project.
+
+Since many members are often a part of one or more projects in the foundation, feel free to discuss it with the members 
+if there is a potential conflict while representing. The representative can either raise an issue and discuss them during 
+the Standards Team meeting.
+
+If a representative has an objection or dissent they should express it as early as possible to ensure 
+there is ample time to discuss and reach consensus before representing Standards Working Group.
+
+Representatives must also conduct themselves in a professional and respectful manner. Some general guidelines include:
+
+* When in doubt always wear the least possible hat.
```

Comment:
not sure what “least” means here; how does one quantify hatness?

Reply:
Is the intent here to convey that you should try to represent the interests of a few parties rather than trying to yourself synthesize the needs of non-foundation projects with those of foundation projects?

Reply:
> not sure what “least” means here; how does one quantify hatness?

Suppose the representative is supporting a project, belongs to an organisation that is having a project, and also doing some JavaScript specification, We do not want that representative to represent everything for their actions. The best possible course in that situation will be deciding to wear the least possible hat. It is very hard to quantify hatness and it is subjective.

> Is the intent here to convey that you should try to represent the interests of a few parties rather than trying to yourself synthesize the needs of non-foundation projects with those of foundation projects?

Roughly that is the intent here.

Reply:
Do you mean, the fewest hats? In other words, a representative should choose one "role" with which to express an opinion instead of invoking multiple at once?

Reply:
> In other words, a representative should choose one "role" with which to express an opinion instead of invoking multiple at once

Yeah, exactly. I will re-phrase it :)